CHANGELOG.rst

```diff
@@ -25,6 +25,20 @@ Security
 ========
 
 
+[3.9.0] - 2021-12-22
+********************
+
+Changed
+=======
+- Changed ``on_interface_created`` to handle to try to also handle as a link up
```

Comment:
```suggestion
- Changed ``on_interface_created`` to try also handle as a link up
```

Reply:
I've just applied the suggestion, thanks.